README.md

```diff
@@ -208,13 +149,6 @@ Feel free to check the [issues page](../../issues/).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] As suggested by the previous reviewer, you're meant to complete the sentence in the show support section, not delete it. The aim is to giver viewers a reason to support your project with starts, to give it more visibility. Kindly add it back

Reply:
![first reviewer](https://user-images.githubusercontent.com/104390780/218942943-33395f25-b37d-4bd6-b5b1-ee07219d2dfe.png)
He actually said i have the option of deleting it if i dont have any for now, which i dont have . So i deleted it. Are you of the opinion that he was wrong? 

Pleasae see the image attached

Reply:
@ Shedrack-Sunday, I left a comment on your comment. I had to come back to use the @Shedrack-Sunday just now because i am not sure you will be notified without it. Awaiting your respsonse. Thank you.

Reply:
Hello @Bestbynature, it was an oversight. However, he made a mistake, as only the FAQ should be optional in your README. Please add it back, and you'll get your project approved. Apologise for that 👏

Reply:
I perfectly understand you but i have this to say. As you are already. You are the third reviewer on my work. This means i have exhausted all the three slots for this project. Now, given that this mistake is not from me... Am i still supposed to pay for this by requesting an extra review ?? What if i need the extra review later in module. 

I would like to know if I can be approved like this? Or if i need to talk with Student Success so they can revert one review slot for me to do this. 

Please what do you think?

Reply:
Hello @Shedrack-Sunday, @Gambit142, @NahnahAJ  I dont know if you have been notified of my response. Please, i need your recommendation about whether i reach out to student success to release one slot of review for me to accommodate the error the last reviewer is trying to correct since he believes it's an oversight from the ffirst reviewer in person of @Gambit142

Reply:
Hello @Bestbynature, please use this [Form](https://airtable.com/shrQAqnBwek5a0O0s) to ask for a second opinion on the review. I understand your please perfectly .

Best regards

Reply:
Thanks @Shedrack-Sunday for the link to the form. A senior reviewer reached out already.